parallel_algorithms/P3179/P3179.md

```diff
@@ -4069,7 +4076,7 @@ template<@_execution-policy_@ Ep, @_sized-random-access-range_@ R, @_sized-rando
 it is called `OR`. For the new overloads we call it `OutR`. We should probably align the names.
 The same question applies to the names `S2` and `OutS` of the sentinel-for-the-output parameter.]{.draftnote audience="LWG"}
 
-[x]{.pnum} Let `result_last` be `result + (last - first)` for the overloads with no parameter `result_last` or `result_r`.
+[x]{.pnum} Let `result_last` be `result + (last - first)` for the overload in namespace `std`.
 
 [x]{.pnum} Let *`N`* be `min(last - first, result_last - result)`.
```

Comment:
Here "min" should also go out of the code block, and perhaps similarly in many (all?) other places.

Reply:
Yep, I think, everywhere

Reply:
Done. Also for `mismatch`, `equal`, `adjacent_find`.

Reply:
Done everywhere.